pkg/providers/v1/variant/fargate/fargate.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"strings"
 
+	awsv2 "github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
nit: i see that the aws is only use for converting pointer to values and vice versa, can we remove that usage and such that we need not use awsv2